data/en/census_household.json

```diff
@@ -1095,25 +1095,12 @@
                     "id": "marital-status",
                     "questions": [{
                         "id": "marital-status-question",
-                        "title": "On 9 April 2017, what is your legal marital or same-sex civil partnership status?",
+                        "title": "On 9 April 2017, what is <em> {{[answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name_possessive }} </em> legal marital or same-sex civil partnership status?",
```

Comment:
remove space by <em> tags

Reply:
@shahi645 Can you check in pre-prod once? I think it has always been 9 April.

Reply:
After checking with Andrew and GDS guidelines 9 is right not 9th.